### PR TITLE
Feat - added card component

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
* Added card variants 
* Deploy preview : https://deploy-preview-11--fast-ui.netlify.app/
Scroll Down To see the ScreenShots

### CODE REVIEW

**Comment by Mohasin-Haque** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /pages/card.html
Browser:   Chrome 97.0.4692.99 on Windows 10
Viewport:  1366 x 649 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-11--fast-ui.netlify.app%2Fpages%2Fcard.html&os=Windows&os_version=10&browser=Chrome&browser_version=97.0&resolution=1366x649&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-11--fast-ui.netlify.app/pages/card.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61fed31feca05f00081bd67e&commentID=61fed3eacab9007f37f87cb3&resolution=resolved)

File: styles/card.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  justify-content: flex-end;
+  gap: 2.5rem;
+  margin-right: 1.5rem;
+  margin-bottom: 1rem;
+  cursor: pointer;
+}
+
+.fas {
+  color: var(--text-color-black);
+}
```
Comment (sharmayatendra) [inline styles/card.css:68]: what is fas?

File: styles/card.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  position: relative;
+  width: 100%;
+}
+
+/* Original image */
+.overlay-image .image {
+  display: block;
+  width: 100%;
+  height: auto;
+}
```
Comment (sharmayatendra) [inline styles/card.css:88]: you can directly target specified class

File: pages/card.html [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <div class="badge_container-main flex-center">
+            <section>
+                <div class="card-container">
+                    <div class="image-div">
+                        <div class="overlay-image">
+                            <img class="card-image" src="../images/card.jpg" alt="error" />
```
Comment (sharmayatendra) [inline pages/card.html:209]: can give specific alt attribute error is not that clear if  by chance image not loaded

**Review by sharmayatendra — COMMENTED** [review]
can work on suggested changes e;lse looks fine to me

File: pages/card.html [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+                    </div>
+                    <footer>
+                        <div class="card-actions">
+                            <button class="card-btn">Buy</button>
+                            <button class="card-btn">Add To Cart</button>
+                            <i class="fas fa-heart"></i>
+                            <i class="fas fa-share-alt"></i>
+                            <i class="fas fa-ellipsis-v"></i>
+                        </div>
+                    </footer>
```
Comment (bishnoimukesh) [inline pages/card.html]: don't use footer tag inside card component

File: pages/card.html [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
```
Comment (bishnoimukesh) [inline pages/card.html]: same for this

File: pages/card.html [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+                            Give yourself a cool look by this cool t-shirt. Made with pure cotton fabrics.
+                        </p>
+                    </div>
+                    <footer>
+                        <div class="card-actions">
+                            <i class="fas fa-heart"></i>
+                            <i class="fas fa-share-alt"></i>
+                            <i class="fas fa-ellipsis-v"></i>
+                        </div>
+                    </footer>
```
Comment (bishnoimukesh) [inline pages/card.html]: same for this

File: pages/card.html [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+                    </div>
+                    <footer>
+                        <div class="card-actions">
+                            <button class="card-btn">Buy</button>
+                            <button class="card-btn">Add To Cart</button>
+                            <i class="fas fa-heart"></i>
+                            <i class="fas fa-share-alt"></i>
+                            <i class="fas fa-ellipsis-v"></i>
+                        </div>
+                    </footer>
```
Comment (bishnoimukesh) [inline pages/card.html]: same for this

File: pages/card.html [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
```
Comment (bishnoimukesh) [inline pages/card.html]: same

File: pages/card.html [hunk 9]
```diff
@@ -0,0 +1,349 @@
...
```
Comment (bishnoimukesh) [inline pages/card.html]: same

File: styles/card.css (hunk 1 above)
Comment (Mohasin-Haque) [reply styles/card.css:68]: Its a class used in icons

**Review by bishnoimukesh — APPROVED** [review]
changes suggested to made. make them and move

File: pages/card.html (hunk 4 above)
Comment (Mohasin-Haque) [reply pages/card.html]: Done!!

File: pages/card.html (hunk 5 above)
Comment (Mohasin-Haque) [reply pages/card.html]: Done!!

File: pages/card.html (hunk 6 above)
Comment (Mohasin-Haque) [reply pages/card.html]: Done!!

File: pages/card.html (hunk 3 above)
Comment (Mohasin-Haque) [reply pages/card.html:209]: OK!

File: pages/card.html (hunk 7 above)
Comment (Mohasin-Haque) [reply pages/card.html]: Done!!

**Pull request merged**